packages/oslo-generator-json-webuniversum/lib/JsonWebuniversumGenerationService.ts

```diff
@@ -0,0 +1,197 @@
+import {
+  ServiceIdentifier,
+  type IService,
+  type Logger,
+  QuadStore,
+  getApplicationProfileLabel,
+  getVocabularyLabel,
+  getVocabularyDefinition,
+  getApplicationProfileDefinition,
+  getVocabularyUsageNote,
+  getApplicationProfileUsageNote,
+  ns,
+  getMinCount,
+  getMaxCount,
+  getCodelist
+} from "@oslo-flanders/core";
+import { inject, injectable } from "inversify";
+import { JsonWebuniversumGenerationServiceConfiguration } from "./config/JsonWebuniversumGenerationServiceConfiguration";
```

Comment:
Maybe change these relative paths to an absolute one to be in line with other imports!